README.md

```diff
@@ -1,101 +1,106 @@
-# Deploy to iTunes Connect with Deliver
```

Comment:
It is weird that the title of the step doesn't contain the deliver.

Reply:
It does for the application loader

https://github.com/bitrise-steplib/steps-deploy-to-itunesconnect-application-loader/blob/master/step.yml#L1

Reply:
I think we can change that without breaking anything, right?